examples/compiled/embedded_csv.vg.json

```diff
@@ -0,0 +1,126 @@
+{
+    "$schema": "https://vega.github.io/schema/vega/v3.0.json",
+    "autosize": "pad",
+    "padding": 5,
+    "style": "cell",
+    "data": [
+        {
+            "name": "source_0",
+            "values": "a\n1\n2\n3\n4",
```

Comment:
So Vega actually supports CSV as inline value too?

Reply:
Yes. This example works.